README.md

```diff
@@ -34,86 +36,85 @@ $ dockle [YOUR_IMAGE_NAME]
   - [Windows](#windows)
   - [Binary](#binary)
   - [From source](#from-source)
+  - [Use Docker](#use-docker)
 - [Checkpoint Summary](#checkpoint-summary)
 - [Quick Start](#quick-start)
   - [Basic](#basic)
   - [Docker](#docker)
 - [Examples](#examples)
-  - [Scan an image](#scan-an-image)
-  - [Scan an image file](#scan-an-image-file)
-  - [Save the results as JSON](#save-the-results-as-json)
-  - [Specify exit code](#specify-exit-code)
-  - [Ignore the specified checkpoints](#ignore-the-specified-checkpoints)
-  - [Clear image caches](#clear-image-caches)
+  - Scan an image
+  - Scan an image file
+  - Get or Save the results as JSON
+  - Specify exit code
+  - Ignore the specified checkpoints
+  - Clear image caches
 - [Continuous Integration](#continuous-integration-ci)
```

Comment:
It seems this link has broken.

Reply:
Sorry, as I mentioned, a lot of links in the TOC are broken I think... 😅
I'll PR if found any!